pitest-maven/src/main/java/org/pitest/maven/PitMojo.java

```diff
@@ -47,7 +47,7 @@
    * @parameter expression="${targetClasses}"
    *
    */
-  protected ArrayList<String>         targetClasses;
+  private List<String>         targetClasses;
```

Comment:
Indeed - was planning on merging that in when I'd had chance to look properly at the very slow site report tests. If you could update the PR to add and ignore to them I'll click the merge button now.